spec/factories/user.rb

```diff
@@ -3,6 +3,6 @@
     first_name "Sname"
     last_name "Emmanuel"
     email "sname@osiname.com"
-    password_digest "password"
+    password "password"
   end
 end
```

Comment:
Sure. It's possible, and might even be a better implementation for testing. Just did. Thanks